<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_prod_id">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_prod_pwd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_login</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>btn_main_to_second()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_main_to_second()</slot>
 </slots>
</ui>
